docker/backend/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN set -x \
         bash \
         curl \
         gcc \
-        python3-pip \
+        python3-distutils \
+        python3 -m pip _rest_of_the_pip \
```

Comment:
This is a typo. _rest_of_the_pip is undefined and I think was copied from a forum question accidentally. From here: https://stackoverflow.com/questions/69503329/pip-is-not-working-for-python-3-10-on-ubuntu.

Reply:
Yep. That's what I did. Will fix.